PLATFORMS.md

```diff
@@ -0,0 +1,63 @@
+
+Several parts of this policy require providing target-specific documentation. Such documentation should typically appear in a subdirectory of the platform-support section of the `liboqs` manual, with a link from the target's entry in platform support.
+
+Note that a target must have already received approval for the next lower tier, and spent a reasonable amount of time at that tier, before making a proposal for promotion to the next higher tier; this is true even if a target meets the requirements for several tiers at once. This policy leaves the precise interpretation of "reasonable amount of time" up to the approving team; the team may scale the amount of time required based on their confidence in the target and its demonstrated track record at its current tier. At a minimum, multiple stable releases of `liboqs` should typically occur between promotions of a target.
+
+The availability or tier of a target in stable `liboqs` is not a hard stability guarantee about the future availability or tier of that target. Higher-level target tiers are an increasing commitment to the support of a target, and we will take that commitment and potential disruptions into account when evaluating the potential demotion or removal of a target that has been part of a stable release. The promotion or demotion of a target will not generally affect existing stable releases, only current development and future releases.
+
+In this policy, the words "must" and "must not" specify absolute requirements that a target must meet to qualify for a tier. The words "should" and "should not" specify requirements that apply in almost all cases, but for which the approving teams may grant an exception for good reason. The word "may" indicates something entirely optional, and does not indicate guidance or recommendations. This language is based on [IETF RFC 2119](https://datatracker.ietf.org/doc/html/rfc2119).
+
+## Platforms supported
+
+### Tier 1
+
+- x86_64/amd64/x64 for Ubuntu Linux (Focal) and MacOS (XCode 14)
+- aarch64 for Ubuntu (Focal)
+- armhf/ARM7 and aarch64 emulation on Ubuntu
+- ppc64le for Ubuntu (Focal)
+- s390x for Ubuntu (Focal)
+
+### Tier 2
+
+- x86_64/amd64/x64 for Windows 2022
+- armeabi-v7a, arm64-v8a, x86, x86_64 for Android
+- OS64, TVOS for Apple Mobile
```

Comment:
I don't know what OS64 is?

Reply:
Me neither. It's [this test run](https://github.com/open-quantum-safe/liboqs/actions/runs/6776860846/job/18419274742) triggered by https://github.com/open-quantum-safe/liboqs/blob/aeac3a441dd3d95821b04a26baae6f73013538d8/.github/workflows/apple.yml#L12 contributed in https://github.com/open-quantum-safe/liboqs/pull/1546. @res0nance could you clarify?.

Reply:
OS64 is ARM64 for iphone